docs/privacy-policy.md

```diff
@@ -0,0 +1,74 @@
+# Privacy Policy
+
+#### _Effective:_ February 8, 2022
+
+
+This policy document explains how the Operate First Data Science Community Meetup (“we” / “us” / “our”) collects, stores, and uses attendees’ personal information. If you wish to access any particular section of this Privacy Policy, use the list below to jump to that section.
+
+1. [Collection Of Information](#Collection-Of-Information)
+
+2. [Use Of Information](#Use-Of-Information)
+
+3. [Sharing Of Information](#Sharing-Of-Information)
+
+4. [Additional Details](#Additional-Details)
+
+5. [References](#References)
+
+6. [Contact Us](#Contact-Us)
+
+
+
+## Collection Of Information
+
+We collect information about you when you:
+
+1. Login using a google account;
+2. Use the google meet platform;
+3. Attend an event;
+4. Visit our website; and
+5. Communicate with us.
```

Comment:
Shall we add the means of communication? is it emails, google chat, slack or more?

Reply:
Yes, "communicate with us" is vague. What does it mean?

Reply:
The list structure here can be confusing. We don't collect information when someone logs in to their gmail or uses the google meet platform. We only collect information when they attend the DS meetup event. 
So maybe just three points
1) Attend DS community meetup through google meet platform
2) Visit our website
3) Communicate using ....
WDYT?

Reply:
Sounds good, we have rephrased this section :+1: